### PR TITLE
OSDOCS-2869: Migrating common SD content in Welcome book

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-2869

### CODE REVIEW

**Review by skrthomas — COMMENTED** [review]
LGTM 💯 A++

File: welcome/index.adoc [hunk 1]
```diff
@@ -54,6 +68,8 @@ endif::openshift-origin[]
 - **Install a cluster on {rh-openstack-first}**: You can install a cluster on
 xref:../installing/installing_openstack/installing-openstack-installer-custom.adoc#installing-openstack-installer-custom[{rh-openstack} with customizations].
 
+You can install a cluster on xref:../installing/installing_openstack/installing-openstack-user.adoc#installing-openstack-user[{rh-openstack} with customizations], xref:../installing/installing_openstack/installing-openstack-user-kuryr.adoc#installing-openstack-user-kuryr[with network customizations], or xref:../installing/installing_openstack/installing-openstack-user-sr-iov.adoc#installing-openstack-user-sr-iov[with SR-IOV] on user-provisioned infrastructure.
```
Comment (bmcelvee) [inline welcome/index.adoc:71]: Pulling into main based on: https://github.com/openshift/openshift-docs/pull/36976/files#diff-40d72ccc658b0f56a1a01387cffdf2a793c5cd21234a0fcbfcc25cf880d318c5R68.

File: welcome/index.adoc [hunk 2]
```diff
@@ -129,15 +150,11 @@ to see your applications, monitor status, connect and group components, and modi
...
 
-ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
+ifdef::openshift-enterprise,openshift-origin[]
 - **Deploy Helm charts**:
-xref:../applications/working_with_helm_charts/understanding-helm.adoc#understanding-helm[Helm 3]
-is a package manager that helps developers define, install, and update
-application packages on Kubernetes. A Helm chart is a packaging format that
-describes an application that can be deployed using the Helm CLI.
-endif::[]
+xref:../applications/working_with_helm_charts/understanding-helm.adoc#understanding-helm[Helm] is a software package manager that simplifies deployment of applications and services to OpenShift Container Platform clusters. Helm uses a packaging format called charts. A Helm chart is a collection of files that describes the OpenShift Container Platform resources.
```
Comment (bmcelvee) [inline welcome/index.adoc:157]: Updated based on https://github.com/openshift/openshift-docs/pull/36976

File: welcome/index.adoc [hunk 3]
```diff
@@ -35,10 +49,14 @@ You can also deploy a cluster on AWS infrastructure that you provisioned yoursel
 
 - **xref:../installing/installing_azure/installing-azure-account.adoc#installing-azure-account[Install a cluster on Azure]**: You can deploy clusters with xref:../installing/installing_azure/installing-azure-default.adoc#installing-azure-default[default settings], xref:../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-customizations[custom Azure settings], or xref:../installing/installing_azure/installing-azure-network-customizations.adoc#installing-azure-network-customizations[custom networking settings] in Microsoft Azure. You can also provision {product-title} into an xref:../installing/installing_azure/installing-azure-vnet.adoc#installing-azure-vnet[Azure Virtual Network] or use xref:../installing/installing_azure/installing-azure-user-infra.adoc#installing-azure-user-infra[Azure Resource Manager Templates] to provision your own infrastructure.
 
+- **xref:../installing/installing_azure_stack_hub/preparing-to-install-on-azure-stack-hub.adoc#preparing-to-install-on-azure-stack-hub[Install a cluster on Azure Stack Hub]**: You can install {product-title}on Azure Stack Hub on user-provisioned infrastructure.
```
Comment (bmcelvee) [inline welcome/index.adoc:52]: Updated based on #36976

File: welcome/index.adoc [hunk 4]
```diff
@@ -35,10 +49,14 @@ You can also deploy a cluster on AWS infrastructure that you provisioned yoursel
...
+
 - **xref:../installing/installing_gcp/installing-gcp-account.adoc#installing-gcp-account[Install a cluster on GCP]**: You can deploy clusters with xref:../installing/installing_gcp/installing-gcp-default.adoc#installing-gcp-default[default settings] or xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-customizations[custom GCP settings] on Google Cloud Platform (GCP). You can also perform a GCP installation where you provision your own infrastructure.
 
 - **xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc#installing-vsphere-installer-provisioned[Install a cluster on VMware vSphere]**: You can install {product-title} on supported versions of vSphere.
 
+- **xref:../installing/installing_vmc/preparing-to-install-on-vmc.adoc#preparing-to-install-on-vmc[Install a cluster on VMware Cloud]**: You can install OpenShift Container Platform on supported versions of VMware Cloud (VMC) on AWS.
```
Comment (bmcelvee) [inline welcome/index.adoc:58]: Updated based on #36976

**Comment by pneedle-rh** [comment]
@bmcelvee I tested a migration build today that added this PR on to the feature branch created by the migration scripts. The  "Developer activities" section of the welcome pages for OSD and ROSA was not included in the preview. I could not see exactly what changed after our previous successful build test. But, on review of `welcome/index.adoc` in the PR, it looked like there was an additional instance of the following line, which was not closed out with an `endif`:

`ifndef::openshift-rosa,openshift-dedicated[]`

This meant that when the subsequent `ifdef::openshift-rosa,openshift-dedicated[]` line was cancelled out, because it was within the `ifndef` for those distros.

I have updated the commit in this PR to remove that line. I have checked the welcome pages of the published versions for different distros with the test build and they look ok. I have not yet been able to compare the docs.okd.io welcome page because that is unavailable at the moment. I will check this as soon as possible.

**Comment by bmcelvee** [comment]
> @bmcelvee I tested a migration build today that added this PR on to the feature branch created by the migration scripts. The "Developer activities" section of the welcome pages for OSD and ROSA was not included in the preview. I could not see exactly what changed after our previous successful build test. But, on review of `welcome/index.adoc` in the PR, it looked like there was an additional instance of the following line, which was not closed out with an `endif`:
> 
> `ifndef::openshift-rosa,openshift-dedicated[]`
> 
> This meant that when the subsequent `ifdef::openshift-rosa,openshift-dedicated[]` line was cancelled out, because it was within the `ifndef` for those distros.
> 
> I have updated the commit in this PR to remove that line. I have checked the welcome pages of the published versions for different distros with the test build and they look ok. I have not yet been able to compare the docs.okd.io welcome page because that is unavailable at the moment. I will check this as soon as possible.

Thank you, @pneedle-rh!

**Pull request merged**